molecule/resources/playbooks/verify_linux.yml

```diff
@@ -15,13 +30,39 @@
 
 - name: Ensure the agent is running
   service:
-    name: "{{ vars[agent_type + '_service_name'] }}"
+    name: "{{ service_name }}"
     state: started
   register: result
+  when: (agent_type != 'ops-agent') or (agent_type == 'ops-agent' and lookup('env', 'VERSION').split('.')[0] == '1')
 
 - name: Assert the agent was already running
   assert:
     that: result.changed == false
+  when: (agent_type != 'ops-agent')
+
+- name: Ensure the ops agent fluentbit agent is running
```

Comment:
Again, this is below the abstraction layer of Ops Agent - you are not supposed to know or care which subagents are running, and we are free to swap them out in the future.

Reply:
With https://github.com/GoogleCloudPlatform/ops-agent/pull/132 now merged you can check the status of the `google-cloud-ops-agent` service (once that PR hits the next release).

Reply:
I just pushed a commit that reverts this change. 2.0.1 works well, 2.0.0 will work with this role but the test cases will fail. I think this is okay.